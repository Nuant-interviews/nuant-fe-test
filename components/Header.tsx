import Image from "next/image";

export const Header = () => {
  return (
    <header className="w-full border-b  mx-auto py-2 dark:border-gray-700 flex ">
      <div className="container w-full flex items-left justify-between align-center py-2 px-6">
        <Image
          className="w-auto h-12 text-black rounded-full p-x4 dark:text-white"
          src="/images/logo.png"
          alt="test"
          width={100}
          height={100}
        />
      </div>
    </header>
  );
};
