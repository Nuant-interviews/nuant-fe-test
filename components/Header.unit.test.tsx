import { describe, expect, test } from "vitest";
import { render } from "@testing-library/react";
import { Header } from "./Header";

describe("<Header /> component", () => {
  test("header is rendered properly", () => {
    const { container } = render(<Header />);

    expect(container).toMatchSnapshot();
  });
});
