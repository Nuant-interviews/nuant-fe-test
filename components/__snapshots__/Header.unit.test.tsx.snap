// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`<Header /> component > header is rendered properly 1`] = `
<div>
  <header
    class="w-full border-b  mx-auto py-2 dark:border-gray-700 flex "
  >
    <div
      class="container w-full flex items-left justify-between align-center py-2 px-6"
    >
      <img
        alt="test"
        class="w-auto h-12 text-black rounded-full p-x4 dark:text-white"
        data-nimg="1"
        decoding="async"
        height="100"
        loading="lazy"
        src="/_next/image?url=%2Fimages%2Flogo.png&w=256&q=75"
        srcset="/_next/image?url=%2Fimages%2Flogo.png&w=128&q=75 1x, /_next/image?url=%2Fimages%2Flogo.png&w=256&q=75 2x"
        style="color: transparent;"
        width="100"
      />
    </div>
  </header>
</div>
`;
